MANIFEST.in

```diff
@@ -0,0 +1 @@
+include pyrometheus/mechanisms/*.yaml
```

Comment:
In view of our discussion this morning, I am not sold that we should ship these in Pyrometheus.